### DISCUSSION TITLE
Twilio 번호 구매 없이 연락처 인증 서비스 5분만에 구현하기

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#28] [Blog Comments]
# Twilio 번호 구매 없이 연락처 인증 서비스 5분만에 구현하기

이번 포스팅에선 Twilio를 이용해 Node.js에서 개인 번호를 발급받지 않고, 핸드폰 번호 인증을 매우 간단하게 구현하는 방법에 대해 소개해드리겠습니다.

https://www.peterkimzz.com/phone-validation-service-twilio-in-5-minutes/

**Comment by Modyland** [comment]
안녕하세요..! 질문드립니다 
네이버 API는 법인계정으로만 SMS 이용이 가능하던데 이건 그냥 개인이 이용할 수 있나요?
사용자가 입력하는 번호 데이터는 앱제작자한테도 남는걸까요?

**Reply by peterkimzz** [reply]
에고 확인을 못했네요. 답장이 늦었습니다.

개인이 이용할 수 있고, 유저가 직접 타이핑한 번호를 서버로 받기 때문에, 번호가 앱 제작자한테 남습니다. 받지 않더라도 Twilio API 요청 성공하면 대시보드에도 모두 남습니다.
